bin/strap.sh

```diff
@@ -325,35 +333,38 @@ if [ -n "$STRAP_GITHUB_USER" ]; then
   fi
 
   DOTFILES_URL="git@github.com:$DOTFILES_REPO"
-  log "Checking for remotes on $DOTFILES_URL"
+  logn "Checking for remotes on $DOTFILES_URL..."
   if git ls-remote "$DOTFILES_URL" &>/dev/null; then USER_DOTFILES_EXISTS=1; fi
+  logk
   if [ -n "$USER_DOTFILES_EXISTS" ]; then
-    log "Fetching $DOTFILES_REPO from GitHub:"
+    logn "Fetching $DOTFILES_REPO from GitHub..."
```

Comment:
the problem here is that  if it doesn't exist... .you get 
```
Fetching git@github.com:jtrinklein/dotfiles from GitHub...Cloning to ~/.dotfiles...√√
```

Reply:
Updated 'Cloning...' output to `logdebug`.